.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint Strings
+
+on: push
```

Comment:
let's contain this to the master branch

Reply:
I missed this request. If you need a technical review from l10n, I would involve someone else from my team. I don't think I will add any value to this, :). Let me know if you do.